packages/charts/frontend/sirius-components-charts/src/bar/BarChart.tsx

```diff
@@ -68,7 +68,8 @@ export const BarChart = ({ width, height, chart }: BarChartProps) => {
       selection.selectAll('*').remove(); // Remove existing content.
       const svg = selection
         .attr('viewBox', [0, 0, width, height])
-        .attr('style', `max-width: 100%; max-height: ${height}; height: auto; height: intrinsic;`);
+        .attr('style', `max-width: 100%; max-height: ${height}; height: auto; height: intrinsic;`)
+        .attr('pointer-events', 'none'); // allow selection anywhere in the svg
```

Comment:
Fix selection problem when users were trying to select the bar chart by clicking in its colored bars.